runtime/compiler/control/CompilationThread.cpp

```diff
@@ -1876,7 +1874,7 @@ void TR::CompilationInfo::invalidateRequestsForNativeMethods(J9Class * clazz, J9
       TR_MethodToBeCompiled *next = cur->_next;
       J9Method *method = cur->getMethodDetails().getMethod();
       if (method &&
-          cur->getMethodDetails().getClass() == clazz &&
+          J9_CLASS_FROM_METHOD(method) == clazz &&
```

Comment:
There are a handful of changes like the above, where we go from using getters to VM `J9_` macros that dereference J9Class/J9Method/etc. Is that intended? I looked at a few cases and it looks like the code will never be reached by a server compilation so it's safe right now, but were the getters doing the wrong thing?

Reply:
Exactly, those routines that include `J9_CLASS_FROM_METHOD` are never executed by the server. I changed them from what they used to be in jitaas branch because (1) I wanted fewer modifications to master and (2) it's slightly faster than calling a virtual method.